core/trino-main/src/test/java/io/trino/sql/query/TestFilterInaccessibleColumns.java

```diff
@@ -210,6 +225,21 @@ public void testMaskingAsAdminWithCaseOnNotAccessibleColumn()
                 .matches("VALUES (CAST('CANADA' AS VARCHAR(25)), BIGINT '1', CAST('eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold' AS VARCHAR(152)))");
     }
 
+    @Test
+    public void testMaskingAsInvokerWithCaseOnNotAccessibleColumn()
```

Comment:
NotAccessible -> Inaccessible

"MaskingAsInvoker" -- what does this mean?
do we have RUN AS INVOKER and RUN AS DEFINER masks?